web/source/osk/oskManager.ts

```diff
@@ -38,14 +38,17 @@ namespace com.keyman.osk {
 
     // OSK positioning fields
     userPositioned: boolean = false;
-    width: number;
-    height: number;
```

Comment:
I don't understand why these two lines are deleted as you refer to `this.width` and `this.height` e.g. on line 292.

Reply:
See lines 288 through 302.  These fields are now computed properties.